i18n/en.json

```diff
@@ -3052,6 +3053,7 @@
   "create_post.deactivated": "You are viewing an archived channel with a **deactivated user**. New messages cannot be posted.",
   "create_post.error_message": "Your message is too long. Character count: {length}/{limit}",
   "create_post.file_limit_sticky_banner.admin_message": "Your free plan is limited to {storageGB} of files. New uploads will automatically archive older files. To view them again, you can delete older files or <a>upgrade to a paid plan.</a>",
+  "create_post.file_limit_sticky_banner.messageTitle": "Your free plan is limited to {storageGB} of files.",
   "create_post.file_limit_sticky_banner.non_admin_message": "Your free plan is limited to {storageGB} of files. New uploads will automatically archive older files. To view them again, <a>notify your admin to upgrade to a paid plan.</a>",
```

Comment:
`create_post.file_limit_sticky_banner.admin_message` and `create_post.file_limit_sticky_banner.non_admin_message` should have their first sentence removed here as well to match the changes in defaultMessage up above.